diff --git a/include/config/general.h b/include/config/general.h
--- a/include/config/general.h
+++ b/include/config/general.h
@@ -62,7 +62,8 @@
 #define GEN_8 7
 #define GEN_9 8
 #define GEN_CHAMPIONS GEN_9 + 1
-#define GEN_COUNT GEN_CHAMPIONS + 1
+#define GEN_10 GEN_CHAMPIONS + 1
+#define GEN_COUNT GEN_10 + 1
 // Changing GEN_LATEST's value to a different Generation will change every default setting that uses it at once.
 #define GEN_LATEST GEN_9
 
