diff --git a/include/pokemon.h b/include/pokemon.h
--- a/include/pokemon.h
+++ b/include/pokemon.h
@@ -132,14 +132,13 @@ struct PokemonSubstruct0
 {
     enum Species species:11; // 2047 species.
     enum Type teraType:5; // 30 types.
-    enum Item heldItem:10; // 1023 items.
-    u16 unused_02:6;
+    enum Item heldItem:16; // Preserve the original Emerald 16-bit held-item save field.
     u32 experience:21;
     u32 nickname11:8; // 11th character of nickname.
     u32 unused_04:3;
     u8 ppBonuses;
     u8 friendship;
     u16 pokeball:6; // 63 balls.
     u16 nickname12:8; // 12th character of nickname.
     u16 unused_0A:2;
